core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -74,17 +75,17 @@ private[spark] class HeartbeatReceiver(sc: SparkContext, clock: Clock)
 
   // "spark.network.timeout" uses "seconds", while `spark.storage.blockManagerSlaveTimeoutMs` uses
   // "milliseconds"
-  private val slaveTimeoutMs =
-    sc.conf.getTimeAsMs("spark.storage.blockManagerSlaveTimeoutMs", "120s")
   private val executorTimeoutMs =
-    sc.conf.getTimeAsSeconds("spark.network.timeout", s"${slaveTimeoutMs}ms") * 1000
+    sc.conf.getTimeAsSeconds("spark.network.timeout",
```

Comment:
I meant something like this to match the docs:
```
  private val executorTimeoutMs =
    sc.conf.getTimeAsMs(
      "spark.storage.blockManagerSlaveTimeoutMs",
      s"${sc.conf.getTimeAsSeconds("spark.network.timeout", "120s")}s")
```
Could you also change https://github.com/apache/spark/blob/53c06ddabbdf689f8823807445849ad63173676f/resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala#L637 to use the above pattern?

Reply:
updated, please have a review, thank you @zsxwing @jiangxb1987